static/scss/profile-image-uploader.scss

```diff
@@ -22,6 +22,6 @@
 
   .photo-active-item {
     width: 100%;
-    height: 500px;
+    height: 300px;
```

Comment:
this seems like a better default value, but can we be smarter here? Does it make sense to use a percentage? or a calculation based on the viewport height minus the chrome?